src/foremast/elb/create_elb.py

```diff
@@ -208,3 +221,68 @@ def add_stickiness(self):
                     stickiness_dict[externalport] = policyname
         return stickiness_dict
+
+
+    def configure_load_balancer_attributes(self, json_data):
+        """Configure load balancer attributes such as idle timeout, connection draining, etc
+
+        Args:
+            json_data (json): return data from ELB upsert
+        """
+        env = boto3.session.Session(profile_name=self.env, region_name=self.region)
+        elbclient = env.client('elb')
+
+        elb_settings = self.properties['elb']
+        log.debug('Block ELB Settings Pre Configure Load Balancer Attributes:\n%s', pformat(elb_settings))
+
+        for job in json.loads(json_data)['job']:
+            load_balancer_attributes = {    
+                'CrossZoneLoadBalancing': {
+                    'Enabled': True
+                 },
+                 'AccessLog': {
+                     'Enabled': False,
+                 },
+                 'ConnectionDraining': {
+                     'Enabled': False,
+                 },
+                 'ConnectionSettings': {
+                     'IdleTimeout': 60
+                 }
+            }
+            if elb_settings.get('connection_draining_timeout'):
+                connection_draining_timeout = int(elb_settings['connection_draining_timeout'])
+                log.info("Adding Backend Connection Draining of %d", connection_draining_timeout)
+                load_balancer_attributes['ConnectionDraining'] = {
+                        'Enabled': True,
+                        'Timeout': connection_draining_timeout
+                }
+            if elb_settings.get('idle_timeout'):
+                idle_timeout = int(elb_settings['idle_timeout'])
+                log.info("Adding ELB Idle Timeout of %d", idle_timeout)
+                load_balancer_attributes['ConnectionSettings'] = {
+                        'IdleTimeout': idle_timeout
+                }
+            if elb_settings.get('access_log'):
+                access_log_bucket_name = elb_settings['access_log']['bucket_name']
+                access_log_bucket_prefix = elb_settings['access_log']['bucket_prefix']
+                access_log_emit_interval = int(elb_settings['access_log']['emit_interval'])
+                log.info("Setting ELB AccessLog path to %s/%s writing every %d minutes", access_log_bucket_name, 
+                                                                                         access_log_bucket_prefix,
+                                                                                         access_log_emit_interval)
+                load_balancer_attributes['AccessLog'] = {
+                        'Enabled': True,
+                        'S3BucketName': access_log_bucket_name,
+                        'EmitInterval': access_log_emit_interval,
+                        'S3BucketPrefix': access_log_bucket_prefix
+                }
+                
+            if load_balancer_attributes:
```

Comment:
I dont think the conditional is needed.
Just do the `
log.info(..)
log.debug(...)
elbclient.modify_load_balancer_attributes()`

What you can do, is earlier (when checking those custom attributes are defined), within the log, add `log.info('Setting custom ELB AccessLog path ...')`. (add 'custom' in the log output)